portfolio/src/main/webapp/fetch.js

```diff
@@ -0,0 +1,15 @@
+/** Adds an onclick event listener to the get hello button. */
+const getHelloButton = document.getElementById('get-hello-button');
+document.addEventListener('click', showHello);
+
+/** Fetches a list of quotes from the server and adds one to the page. */ 
+async function showHello() {
+  const responseFromServer = await fetch('/hello');
+  const quotes = await responseFromServer.json();
+
+  // Pick a random quote.
+  const quote = quotes[Math.floor(Math.random() * quotes.length)];
+
+  const getHelloContainer = document.getElementById('get-hello-container');
+  getHelloContainer.innerHTML = quote;
```

Comment:
`innerHTML` can make your site vulnerable to XSS attacks and should be avoided (https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations). Use `innerText` or `textContent` instead.

Reply:
Wow, I did not know that, thank you for the information. I will fix it.